stylesheets/style.css

```diff
@@ -5,3 +5,222 @@ main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0 0px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
```

Comment:
This si not reset anymore, maybe you could add a breaking comment to specify that